cmd/cluster.go

```diff
@@ -0,0 +1,133 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/rancher/rke/hosts"
+	"github.com/rancher/rke/services"
+	"github.com/urfave/cli"
+)
+
+func ClusterCommand() cli.Command {
+	clusterUpFlags := []cli.Flag{
+		cli.StringFlag{
+			Name:   "cluster-file",
+			Usage:  "Specify an alternate cluster YAML file (default: cluster.yml)",
```

Comment:
You can set the default value directly in cli.flag, no need to hardcode

Reply:
done